### PR TITLE
Add member invititation step to roadmap creation

### PR DESCRIPTION
**Pull request by IidaKainu**
Details

- The 'add member' button is displayed at the top while no members are present. Otherwise, it's displayed at the bottom
- The member list is scrollable, with a static add button
 

- a warning tooltip is shown when 'skip it' is clicked with added members present

- If any members were added, a link to people page is included in the last part:

### CODE REVIEW

**Review by juhoeerola — COMMENTED** [review]
The emails could be checked to be unique, as there is no reason for multiple invitations

File: frontend/src/components/modals/modalparts/TeamMemberModalParts.tsx [hunk 1]
```diff
@@ -77,3 +71,133 @@ export const AddTeamMemberInfo: FC<{
...
+        label={t('Member email the link will be sent to')}
+        placeholder={t('Example email', { localPart: 'teammember' })}
+        name="send link"
+        type="email"
+        value={member.email}
+        onChange={(e) => {
+          setMember({ ...member, email: e.currentTarget.value });
+          setValidForm(
+            e.currentTarget.checkValidity() && !!e.currentTarget.value,
+          );
```
[the remark this answers is not in the document]
Comment (IidaKainu) [reply frontend/src/components/modals/modalparts/TeamMemberModalParts.tsx]: I did not think of that, thank you!

**Comment by IidaKainu** [comment]
Added the suggestions

> The emails could be checked to be unique, as there is no reason for multiple invitations

The email is checked on addition or modification and an error is displayed if it is not unique

**Review by juhoeerola — APPROVED** [review]
Seems ok 👍

**Pull request merged**